app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailer < ApplicationMailer
+
+  def registration_success(user)
+    @user = user
+    mail(to: @user.email, subject: 'Welcome to MemryBook')
```

Comment:
registration success template is used for mail templates and i need to pass email there.